views/js/runner/provider/qti.js

```diff
@@ -139,7 +139,11 @@ define([
                 var context = self.getTestContext();
                 var states = self.getTestData().states;
                 if(context.state <= states.interacting){
-                    self.loadItem(context.itemUri);
+                    if (context.isTimeout) {
+                        self.timeout();
```

Comment:
Actually not. The timeout action requests the server, which will send a new context.
This action is called only when the test is still alive (interacting or something approaching), and only when the timeout flag is set. 
When a timeout is triggered, the server will update the context and must not return a new timeout context, it will rather return the next available route or a closed state.